src/slashcommands/Help.ts

```diff
@@ -18,16 +19,22 @@ export class Help implements SlashCommand {
 	options = [];
 	requiredPermissions: bigint[] = [
 		Permissions.FLAGS.SEND_MESSAGES,
-		Permissions.FLAGS.EMBED_LINKS,
+		Permissions.FLAGS.EMBED_LINKS, 
 		Permissions.FLAGS.MANAGE_MESSAGES,
 		Permissions.FLAGS.ADD_REACTIONS,
 	];
 	async run(bot: Bot, interaction: CommandInteraction): Promise<void> {
 		try {
+			type cmdList = {[index:string]: string};
+			let cmds = {} as cmdList;
+			bot.slashCommands.forEach((command)=>{
```

Comment:
I don't exactly like that you're looping through all the commands just to assign them to an object again, we can find the command by name in the array, but I understand that it makes accessing them easier in bulk